modular_skyrat/master_files/code/datums/traits/neutral.dm

```diff
@@ -13,9 +13,9 @@
 
 /datum/quirk/personalspace
 	name = "Personal Space"
-	desc = "You'd rather people keep their hands to themselves, and you won't let anyone touch your ass.."
-	gain_text = span_notice("You'd like it if people kept their hands off your ass.")
-	lose_text = span_notice("You're less concerned about people touching your ass.")
+	desc = "You'd rather people keep their hands to themselves."
```

Comment:
Changing the desc kind of makes it much less apparent what the purpose of the quirk actually is.

Reply:
As mentioned in comments, I think 'You'd rather people keep their hands off your butt.' is about as short and.. "professional" as it can be for what the quirk is and the game/community it's part of. Still has a hint of funny, but less staggering to look at, I suppose.

Reply:
I disagree. But I will change it anyway.